src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+body {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search-bar {
+  box-sizing: border-box;
+  width: 100%;
+  font-family: "Avenir", sans-serif;
```

Comment:
Set font styles to html tag, other elements will inherit it (except placeholders, you have to set `inherit` for them)